service_configuration_lib/spark_config.py

```diff
@@ -119,8 +119,19 @@ def get_mesos_spark_env(
     return spark_env
 
 
-def _generate_volume_name(volume):
-    return volume['containerPath']
+def _generate_volume_name(idx):
+    '''Generate a valid name for k8s volume resource.
+
+    Volume name must be a DNS_LABEL and unique within the pod. More info:
+    https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+
+    In specific, volume name must:
+    * contain at most 63 characters
+    * contain only lowercase alphanumeric characters or ‘-’
+    * start with an alphanumeric character
+    * end with an alphanumeric character
+    '''
+    return idx
```

Comment:
Why do we have a function that literally just returns its argument?  Can't we just delete this and use the idx directly?